Adding docker-compose files (#1659)
pyansys/pymapdl: doc/source/getting_started/docker.rst

    Server Executable   : MapdlGrpc Server
    Server listening on : 0.0.0.0:50052

Using ``docker-compose`` to launch MAPDL

> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

You can also use ``docker-compose`` command to launch MAPDL configured in
a ``docker-compose`` file.
This is useful if you want to load an already configured environment, or
if you want to launch multiple instances of MAPDL or services.

> For you convenience, in the folder `docker <pymapdl_docker_dir_>`_ you 
> can find three already configured ``docker-compose`` files that you can
> use: 
> 

Reviewer: ```suggestion
For you convenience, the directory `docker <pymapdl_docker_dir_>`_ 
contains pre-configured ``docker-compose`` files that you can
use:

```

* `docker-compose.yml <pymapdl_docker_compose_base_>`_: the **base** 
  configuration file which launch a remote instance of MAPDL which you
  can connect to. Similar approach to the one described above.

* `docker-compose.local.yml <pymapdl_docker_compose_base_>`_: 
  This file is an extension of the base configuration file which launch
  an Ubuntu Docker image with MAPDL installed in it. 
  This is useful if you want to run MAPDL locally inside this container
  for example for debugging purposes.
  You can connect your VSCode instance to this container by selecting

>    ``Attach to a running container`` option from the command palette.

Reviewer: ```suggestion
  ``Attach to a running container`` option from the command palette.
```

  For more details visit `Attach to a running container <vscode_attach_to_container_>`_.

* `docker-compose.license_server.yml <pymapdl_docker_compose_license_server_>`_:
  This file is an extension of the base configuration file which launch
  a license server. 
  This is useful if you want to run MAPDL remotely and you don't have 
  access to a license server.
  You need a valid license file to run the license server.
  You can use it together with ``docker-compose.local.yml`` hence you
  can connect to it the same way.
  The call to this docker file should be always the last one in the
  ``docker-compose`` command. For instance:

  .. code:: bash

     docker-compose -f docker-compose.yml -f docker-compose.local.yml -f docker-compose.license_server.yml up
  

.. warning:: About the license server image.
   The license server is not intended to be used in production. 
   It is only intended for testing/debugging purposes.
   Its access is limited to collaborators of the PyAnsys project.

>    If you would like to have access to it, please contact us at 

Reviewer: ```suggestion
   If you would like to have access to it, please contact PyAnsys support at
```

    IMAGE=ghcr.io/pyansys/pymapdl/mapdl:$VERSION
    docker run -e ANSYSLMD_LICENSE_FILE=$LICENSE_SERVER -p 50052:50052 $IMAGE -np 4

For additional command line arguments please see the *Notes* section

> for `launch_mapdl <_pymapdl_launch_mapdl_>`_.

Reviewer: ```suggestion
within `launch_mapdl <_pymapdl_launch_mapdl_>`_.
```
